src/components/Main.js

```diff
@@ -37,15 +38,17 @@ const Main = () => {
     */
   const renderResults = () => testData.map((recipe) => (
     <div>
+      <ListGroup.Item as="li">
```

Comment:
The links in the ListGroup go to an empty page, but I know you just set this up to get the navbar working.